.github/workflows/unit_tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: Run Unit Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        pip install -e . -r requirements_dev.txt
+    
+    - name: Launch tests
+      run: |
+        pytest --cov=checkQC --cov-report=xml tests/
```

Comment:
This is not CheckQC :)  Also, projman_filler has not been configured to use codecov. I think we should skip the codecov pats for now, we can add it later.